### PR TITLE
feat(metrics): Documented metric filters.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@dorbin

### CODE REVIEW

File: setup/monitoring/index.md [hunk 1]
```diff
@@ -51,6 +58,93 @@ dashboards](#supplied-dashboards) to your third-party system of choice.
 
 See also [`hal config metric-stores`](/reference/halyard/commands/#hal-config-metric-stores).
 
+### Configuring metric filters
+
+The daemon has a "metric_filter_dir" property that can point to a directory containing
```
Comment (dorbin) [inline setup/monitoring/index.md:65]: Can you point to the specific yaml file containing this property?

File: setup/monitoring/index.md [hunk 2]
```diff
@@ -31,6 +31,13 @@ You can also use the microservice HTTP endpoint `/spectator/metrics`
 directly to scrape metrics yourself. The JSON document structure is
 further documented in the Monitoring reference section.
 
+Note that there are numerous instrumented places within the codebase, not all
+of which are of interest to operators or non-mega-scale users (e.g. for tuning
+large scale performance issues as the envelope is pushed further). The
```
Comment (dorbin) [inline setup/monitoring/index.md]: I think we might be able to get away with deleting the first sentence, and just get straight to "The daemon can be configured..."

File: setup/monitoring/index.md [hunk 3]
```diff
@@ -31,6 +31,13 @@ You can also use the microservice HTTP endpoint `/spectator/metrics`
...
+daemon can be configured to control which collected metrics are forwarded
+to the persistent metrics store. This can alleviate costs or pressure on the
```
Comment (dorbin) [inline setup/monitoring/index.md]: s/or/and

File: setup/monitoring/index.md [hunk 4]
```diff
@@ -51,6 +58,93 @@ dashboards](#supplied-dashboards) to your third-party system of choice.
 
 See also [`hal config metric-stores`](/reference/halyard/commands/#hal-config-metric-stores).
 
+### Configuring metric filters
+
+The daemon has a "metric_filter_dir" property that can point to a directory containing
+metric filter specifications. The default directory is /opt/spinnaker-monitoring/filters.
```
Comment (dorbin) [inline setup/monitoring/index.md:66]: Recommend mentioning `default.yml` first?

"You can configure filters for all services in `default.yml`, and override the filters per service in `<service>.yml.

**Review by dorbin — COMMENTED** [review]
Per our discussion...is it worth mentioning the stackdriver limitation that makes it kind-of necessary to set up some filters, if you're using Stackdriver?

**Comment by ewiseblatt** [comment]
ping

**Review by dorbin — APPROVED** [review]
LGTM

**Pull request merged**